.github/workflows/lint.yaml

```diff
@@ -29,3 +29,7 @@ jobs:
       - name: Run black
         run: |
           black ./ --check --diff
+
```

Comment:
This would be better from the RamaLama perspective if this build also ran on macOS

Reply:
I agree. Will add this in a follow-up PR here.